zio-http-logging/src/test/scala/zhttp/logging/LoggerSpec.scala

```diff
@@ -0,0 +1,62 @@
+package zhttp.logging
+import zio.test._
+
+import java.nio.file.{Files, Paths}
+
+object LoggerSpec extends DefaultRunnableSpec {
+  private val logFile             = Paths.get("target/file.log")
+  private val consoleLogTransport = LoggerTransport.console
+  private val fileLogTransport    = LoggerTransport.file(logFile)
```

Comment:
Should use an in-memory logger for unit tests.

Reply:
My goal was to implement integration testing.